backend/data/src/main/kotlin/io/tolgee/security/ratelimit/RateLimitPolicy.kt

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2023 Tolgee s.r.o. and contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.tolgee.security.ratelimit
+
+data class RateLimitPolicy(
+  val bucketName: String,
+  val limit: Int,
+  val windowSize: Long,
```

Comment:
We could but it'd be very slightly annoying because it'd mismatch with the config type which is a plain long. Definitely possible though, Duration is serializable.